app/instrumentation-tests/src/org/commcare/androidTests/OtherActionTests.kt

```diff
@@ -0,0 +1,266 @@
+package org.commcare.androidTests
+
+
+import android.view.KeyCharacterMap
+import android.view.KeyEvent
+import android.view.View
+import android.view.ViewGroup
+import android.view.inputmethod.EditorInfo
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.contrib.RecyclerViewActions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import junit.framework.Assert.assertTrue
+import org.commcare.annotations.BrowserstackTests
+import org.commcare.dalvik.R
+import org.commcare.utils.CustomMatchers
+import org.commcare.utils.InstrumentationUtility
+import org.commcare.utils.isPresent
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers
+import org.hamcrest.Matchers.*
+import org.hamcrest.TypeSafeMatcher
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.Description
+import org.junit.runner.RunWith
+import java.util.*
+
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+@BrowserstackTests
+class OtherActionTests: BaseTest() {
+    companion object {
+        const val CCZ_NAME = "other.ccz"
+        const val APP_NAME = "Other Test"
+    }
+
+    @Before
+    fun setup() {
+        installApp(APP_NAME, CCZ_NAME)
+        InstrumentationUtility.login("test1", "123")
+    }
+
+    @Test
+    fun testActionKey() {
+        InstrumentationUtility.openForm(0, 3)
+        onView(withClassName(endsWith("EditText")))
+            .perform(click())
+        assertTrue(KeyCharacterMap.deviceHasKey(KeyEvent.KEYCODE_ENTER))
```

Comment:
this seems to fail on [BS](https://app-automate.browserstack.com/dashboard/v2/builds/f6bd6dcbcf6b333661be7a7d1680f8db7f181af7/sessions/tests/68f0d8a96484d9728b619b80929f81b12007622602e07703)

Reply:
I have removed this entire class file from this PR now as you have mentioned in the other comment.